Thumbprint/Targets/Playground/Classes/Inspectable/InspectableViews/Label+InspectableView.swift

```diff
@@ -57,6 +56,7 @@ extension Label: InspectableView {
         label.text = "This is a label"
         label.setContentCompressionResistancePriority(.required, for: .horizontal)
         label.setContentHuggingPriority(.required, for: .horizontal)
+        label.isUserInteractionEnabled = true
```

Comment:
Why was this change necessary ?

Reply:
As label are by default not interactive. This was preventing it from dragging/dropping in the view.